src/SConscript

```diff
@@ -44,8 +44,11 @@ if sys.platform != 'darwin':
 include = ['#/controller/src', '#/build/include', '#controller/lib']
 
 libpath = ['#/build/lib']
-#WINDOWS-TEMP
-libs = ['boost_system', 'log4cplus', 'libboost_system-vc140-mt-gd-1_62', 'libboost_date_time-vc140-mt-gd-1_62', 'libboost_regex-vc140-mt-gd-1_62', 'libboost_program_options-vc140-mt-gd-1_62', 'libboost_thread-vc140-mt-gd-1_62', 'libboost_filesystem-vc140-mt-gd-1_62']
+
+if sys.platform.startswith('win'):
```

Comment:
Sometimes you decide the platform via `if platform.system() == 'Windows'` and sometimes `if sys.platform.startswith('win')`. Please decide on one.